dubbo-filter/dubbo-filter-validation/src/main/java/org/apache/dubbo/validation/support/jvalidation/JValidator.java

```diff
@@ -275,30 +275,30 @@ public void validate(String methodName, Class<?>[] parameterTypes, Object[] argu
             groups.add(1, clazz);
 
             // convert list to array
-            Class<?>[] classgroups = groups.toArray(new Class[groups.size()]);
+            Class<?>[] classGroups = groups.toArray(new Class[groups.size()]);
 
             Object parameterBean = getMethodParameterBean(clazz, method, arguments);
             if (parameterBean != null) {
-                violations.addAll(validator.validate(parameterBean, classgroups ));
+                violations.addAll(validator.validate(parameterBean, classGroups));
             }
 
             for (Object arg : arguments) {
-                validate(violations, arg, classgroups);
-            }
-
-            if (!violations.isEmpty()) {
-                logger.info("Failed to validate service: " + clazz.getName() + ", method: " + methodName + ", cause: " + violations);
-                throw new ConstraintViolationException("Failed to validate service: " + clazz.getName() + ", method: " + methodName + ", cause: " + violations, violations);
+                validate(violations, arg, classGroups);
             }
         } catch (ValidationException e) {
             // only use exception's message to avoid potential serialization issue
             throw new ValidationException(e.getMessage());
```

Comment:
`javax.validation.ConstraintViolationException` might have potential serialization issue. Should be converted.